charts/rancher-external-dns/v0.1.3/values.yaml

```diff
@@ -1,6 +1,6 @@
 ## Details about the image to be pulled.
 image:
-  repository: rancher/kubernetes-external-dns
+  repository: rancher/mirrored-kubernetes-external-dns
   tag: v0.7.3
   pullSecrets: []
   pullPolicy: IfNotPresent
```

Comment:
Technically we need to add questions.yaml to all of these previous external-dns charts and max them at 2.6, that way the old image won't get listed for airgap.